pkg/wallet/account/account.go

```diff
@@ -0,0 +1,246 @@
+package account
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"errors"
+	"fmt"
+
+	"github.com/awnumar/memguard"
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/massalabs/station-massa-wallet/pkg/crypto"
+	"github.com/massalabs/station-massa-wallet/pkg/types"
+	"github.com/massalabs/station-massa-wallet/pkg/types/object"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	AccountLastVersion    = 1
+	AccountUnknownVersion = 0
+)
+
+var (
+	ErrInvalidPrivateKey = errors.New("invalid private key")
```

Comment:
Why the private key is having it's own error? What about reusing the InvalidParameter aggregation instead?